index.html

```diff
@@ -4580,6 +4580,319 @@ <h1>Current Issues</h1>
+    <a>DID controller</a> that describes:
+    </p>
+    <ul>
+        <li>
+        The <a>DID subject</a> itself (e.g., the <code><a>id</a></code> and
+        <code><a>alsoKnownAs</a></code> properties)
+        </li>
+        <li>
+        How to interact with the <a>DID subject</a> (e.g., the
+        <code><a>verificationMethod</a></code> and <code><a>service</a></code>
+        properties).
+        </li>
+        <li>
+        How to to interpret the specific representation of the <a>DID document</a>
+        (e.g., the <code>@context</code> property for a JSON-LD representation).
+        </li>
+    </ul>
+    <p>
+    There is only one required property in a <a>DID document</a>—the <code><a>id</a></code>
+    property—so that is the only statement guaranteed to be in a <a>DID document</a>.
+    That statement is illustrated by the solid red arrow in figure 2 asserting
+    that the <a>DID</a> identifies the <a>DID subject</a>.
+    </p>
+
+    <h2>How can you discover more information about the DID subject?</h2>
+    <p>
+    This is the purpose of the <code><a>alsoKnownAs</a></code> property. The
+    <a>DID controller</a> can use it to provide a list of other URIs (including
+    <a>DIDs</a>) that identify the same <a>DID subject</a>. Resolving or
+    dereferencing these URIs may yield other descriptions or representations of
+    the <a>DID subject</a> as illustrated in figure 3.
+    </p>
+        <figure id="alsoKnownAs-graph">
+          <img style="margin: auto; display: block; width: 75%;"
+            src="diagrams/figure-a.2-also-known-as-graph.png" alt="
+            Diagram showing a graph model that adds to figure 2 by showing an
+            alsoKnownAs property with an arc to another node representing a
+            different resource that dereferences to another description of the
+            DID subject.
+          " >
+          <figcaption>
+              A <a>DID document</a> can use the alsoKnownAs property to assert
+              another URI (including another <a>DID</a>) that identifies the
+              same <a>DID subject</a>
+          </figcaption>
+        </figure>
+    <p>
+    This mechanism is how DID identification can fulfill a longstanding
+    recommendation from the W3C in <a href="https://www.w3.org/TR/cooluris/">Cool
+    URIs for the Semantic Web</a>:
+    </p>
+    <blockquote>
+        Given only a URI, machines and people should be able to retrieve a
+        description about the resource identified by the URI from the Web. Such
+        a look-up mechanism is important to establish shared understanding of
+        what a URI identifies. Machines should get RDF data and humans should
+        get a readable representation, such as HTML.
+    </blockquote>
+    <p>
+    Note that, although it is not required that a <a>DID document</a> use an
+    RDF-based representation such as JSON-LD, that format would meet the letter
+    of this W3C recommendation.
```

Comment:
```suggestion
    Note that it is not required that a <a>DID document</a> use an
    RDF-based representation; see <a href="#representations"></a>.
```

"this W3C recommendation" is ambiguous - unclear if it refers to Cool URIs or DID Core. Also agree with Ivan's comment - Cool URIs is *not* a REC.

Reply:
@rhiaro Totally agree; suggestion taken.